Tools/BasicStation-Certificates-Generation/.gitignore

```diff
@@ -0,0 +1,3 @@
+ca
```

Comment:
Since we spent some effort consolidating these `.gitignores` previously, should we add this to the top-level `.gitignore`?

Reply:
Done, thanks